ui-manchette/src/styles/manchette.css

```diff
@@ -1,5 +1,44 @@
 .manchette-container {
+  .menu {
+    position: absolute;
+    z-index: 10;
+    @apply bg-grey-5;
+    width: 305px;
+    top: 30px;
+    left: 0;
+    border: 0.0625rem solid #b6b2af;
+    border-radius: 0.5rem;
+    box-shadow:
+      0 0.375rem 1.3125rem -0.3125rem rgba(101, 169, 232, 0.341810533216783),
+      0 1rem 1.875rem -0.3125rem rgba(0, 0, 0, 0.2),
+      0 0.1875rem 0.3125rem -0.125rem rgba(0, 0, 0, 0.1),
+      inset 0 0 0 0.0625rem #ffffff;
+
+    /* .menu-item { */
+    button {
+      width: 100%;
+      height: 2.75rem;
+      font-size: 0.875rem;
+      display: flex;
+      align-items: center;
+
+      &:not(:first-child) {
+        border-top: 0.0625rem solid #ffffff;
+      }
+
+      &:not(:only-of-type, :last-child) {
+        border-bottom: 0.0625rem solid #d3d1cf;
+      }
+
+      /* .icon { */
+      span:first-child {
```

Comment:
Should we use the class name here instead? Any reason why `:first-child` is used?

Reply:
Because I went to the full customize option for the waypoint menu. This would mean the user needs to pass more props for each `className` of the menu, it seemed pretty complicated to me.

Reply:
Hm, that sounds like a somewhat awkward middle ground to me: on one hand the structure of the menu is entirely managed by osrd-ui, on the other hand all of the styling is performed outside.

What's the use-case for keeping the styling outside? The menu appearance needs to be consistent with the manchette's styling, so I'm not sure I understand the motivation here?

If we really want to keep the styling out of osrd-ui, why do we need to have the menu and menu item components inside osrd-ui? We could pass the menu component from the outside as a prop.

Reply:
I honestly don't have any passion to keep this at it is now. I would even agree that it would be nice for a team to have a fully functional waypoint menu if they wanted it. I just went that way because for my understanding it had been decided that the spacetime/manchette had to remain as neutral as possible.

But I agree with you that right now we are in a middle ground :)